client/src/index.css

```diff
@@ -31,3 +31,8 @@ code {
   height: 60vh;
   max-height: 100%;
 }
+
+#claim-button {
+  background-color: #7ed957;
+  margin-left: 3px;
+}
```

Comment:
So this overwrites the bootstrap styling on the button and personally I think it looks better without it so I'd get rid of this. Currently it's a different shade of green all the other buttons.

Reply:
before it was a kinda ugly grey i believe

Reply:
Oh but this styling is applying to the confirm button not the go back button.

Reply:
good point i htought i had it the other way